File: apps/parser-worker-python/src/queries/resume_queries.py
"""Resume-related SQL queries for Parser worker."""

UPDATE_RESUME_PARSED_DATA = """
    UPDATE resumes
    SET
        name = %s,
        email = %s,
        phone = %s,
        skills = %s,
        experience = %s::jsonb,
        education = %s::jsonb,
        parsed_data = %s::jsonb,
        updated_at = NOW()
    WHERE id = %s
"""

UPDATE_RESUME_STATUS = """
    UPDATE resumes
    SET status = %s, updated_at = NOW()
    WHERE id = %s
"""

File: apps/export-worker-node/src/queries/resume.queries.ts
/** Resumes SQL queries for export worker */

const RESUME_EXPORT_SELECT = `id, file_name, name, email, phone, skills, experience, education, status, created_at`;

export const GET_RESUMES_FOR_EXPORT_ALL = `
  SELECT ${RESUME_EXPORT_SELECT}
  FROM resumes
  ORDER BY created_at DESC
`;

export const GET_RESUMES_BY_IDS = `
  SELECT ${RESUME_EXPORT_SELECT}
  FROM resumes
  WHERE id = ANY($1)
  ORDER BY created_at DESC
`;
